packages/url_launcher/url_launcher_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.0-nullsafety
```

Comment:
@amirh do you have any comment on this version bump?

Reply:
I think that for the null safety migration as we expect all platform implementations to get the same bump in close proximity it is reasonable to do a major bump on the platform interface.